src/bridge/IrcHandler.ts

```diff
@@ -448,6 +448,55 @@ export class IrcHandler {
         }
```

Comment:
@ara4n mentioned that we should add some state event when the room is unbridged, and another for when it comes back to allow clients to show the disconnect and for us to debug it.